Code/Legacy/CryCommon/StringUtils.h

```diff
@@ -65,7 +65,7 @@ namespace CryStringUtils_Internal
     template <class TChar>
     inline bool strcpy_with_clamp(TChar* const dst, size_t const dst_size_in_bytes, const TChar* const src, size_t const src_size_in_bytes)
     {
-        COMPILE_TIME_ASSERT(sizeof(TChar) == sizeof(char) || sizeof(TChar) == sizeof(wchar_t));
+        static_assert(sizeof(TChar) == sizeof(char) || sizeof(TChar) == sizeof(wchar_t));
```

Comment:
Since the other PR https://github.com/o3de/o3de/pull/2757/files# is deleting all of CryString StringUtils.h will no longer exist, so if that goes in first, you will have a merge conflict with yourself.

Reply:
yeah, will have to merge it, no big deals, this file was deleted and `COMPILE_TIME_ASSERT` was a search/replace